index.html

```diff
@@ -4,22 +4,26 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <title>My Portfolio Home Page</title>
     <link rel="stylesheet" href="./src/css/styles.css" type="text/css" />
   </head>
   <body>
-    <header>
+    <header id="portfolio-section">
       <nav class="nav-bar">
         <ul class="list-logo">
           <li class="logo">My Logo</li>
         </ul>
-        <ul class="list-menu">
-          <li class="menu-icon">
-            <img src="./src/assets/images/menu-hamburger-blue.svg" alt="menu icon" />
-          </li>
-          <li class="menu-item"><a href="#">Portfolio</a></li>
-          <li class="menu-item"><a href="#">About</a></li>
-          <li class="menu-item"><a href="#">Contact</a></li>
+        <button
+          class="menu-toggle"
+          aria-expanded="false"
+          aria-controls="list-menu"
+        >
+          <span class="sr-only">Menu</span>
+        </button>
+        <ul id="list-menu" data-visible="false" class="list-menu">
+          <li class="menu-item"><a href="#portfolio-section">Portfolio</a></li>
+          <li class="menu-item"><a href="#about-section">About</a></li>
+          <li class="menu-item"><a href="#contact-section">Contact</a></li>
         </ul>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure your mobile menu follows your chosen Figma design template(e.g.: you should use the correct **font-weight and line height** on each navigation option, good styling and position of the **CloseMenu** button, the mobile menu to cover the whole page,...)

| Figma Design | Your Design |
| ------------------- | --------------------- |
| ![Screenshot from 2022-09-06 00-56-36](https://user-images.githubusercontent.com/83042742/188518810-c7d1bcf1-fc20-4d0c-8f5b-58977167b174.png)| ![Screenshot from 2022-09-06 00-51-24](https://user-images.githubusercontent.com/83042742/188518709-3fc7065d-727a-4324-be30-da2e15f655ec.png) |